.github/workflows/build_and_package.yaml

```diff
@@ -137,14 +130,104 @@ jobs:
+            sbomName: sbom-controlplane-latest
+            artifactName: controlplane-latest.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/control-plane:${{ github.ref_name }}-amd64
+            sbomName: sbom-controlplane-amd64
+            artifactName: controlplane-amd64.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/control-plane:${{ github.ref_name }}-arm64
+            sbomName: sbom-controlplane-arm64
+            artifactName: controlplane-arm64.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/control-plane-migrations:latest
+            sbomName: sbom-controlplane-migrations-latest
+            artifactName: controlplane-migrations-latest.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/control-plane-migrations:${{ github.ref_name }}-amd64
+            sbomName: sbom-controlplane-migrations-amd64
+            artifactName: controlplane-migrations-amd64.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/control-plane-migrations:${{ github.ref_name }}-arm64
+            sbomName: sbom-controlplane-migrations-arm64
+            artifactName: controlplane-migrations-arm64.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/artifact-cas:latest
+            sbomName: sbom-cas-latest
+            artifactName: cas-latest.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/artifact-cas:${{ github.ref_name }}-amd64
+            sbomName: sbom-cas-amd64
+            artifactName: cas-amd64.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/artifact-cas:${{ github.ref_name }}-arm64
+            sbomName: sbom-cas-arm64
+            artifactName: cas-arm64.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/cli:latest
+            sbomName: sbom-cli-latest
+            artifactName: cli-latest.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/cli:${{ github.ref_name }}-amd64
+            sbomName: sbom-cli-amd64
+            artifactName: cli-amd64.cyclonedx.json
+          - image: ghcr.io/chainloop-dev/chainloop/cli:${{ github.ref_name }}-arm64
+            sbomName: sbom-cli-arm64
+            artifactName: cli-arm64.cyclonedx.json
+    steps:
+      - name: Docker login to Github Packages
+        uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc # v2.2.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install Chainloop
+        run: |
+          curl -sfL https://docs.chainloop.dev/install.sh | bash -s
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: anchore/sbom-action@df80a981bc6edbc4e220a492d3cbe9f5547a6e75 # v0.17.9
+        with:
+          image: ${{ matrix.bundle.image }}
+          format: cyclonedx-json
+          artifact-name: ${{ matrix.bundle.artifactName }}
```

Comment:
what's the artifact name for?

Reply:
It's name of the artifact that will be used when adding it to the release notes. Basically the name it will have there.